docs/developer-guide/ksqldb-reference/pause.md

```diff
@@ -0,0 +1,22 @@
+---
+layout: page
+title: PAUSE
+tagline:  ksqlDB PAUSE statement
+description: Syntax for the PAUSE statement in ksqlDB
+keywords: ksqlDB, query, end, stop
+---
+
+PAUSE
+=========
+
+Synopsis
+--------
+
+```sql
+PAUSE query_id | ALL;
+```
+
+Description
+-----------
+
+Pause a persistent query.  Transient queries cannot be paused or resumed.

```

Comment:
Are there Kafka Streams docs detailing what it actually means to pause a query (e.g., if users have questions about whether their metrics will stop being emitted, whether restore will stop, etc)? If yes, it'd be good to link to those docs from here.

Reply:
Do you have a link to the AK docs? I suspect that's more suitable than the KIP.

Reply:
That's too bad. I can easily see users having all sorts of questions about it actually means to pause a query, and it'd be great if the docs answered them preemptively, but I don't think it's a blocker for getting this docs PR merged. 

I suspect you'll want to write a blurb for the release blog post (whenever this release goes out) about this feature anyway, so you can always augment these docs with similar content at that time.